book/AIDO/15_rules/10_performance.md

```diff
@@ -1,6 +1,8 @@
 # Performance objective {#performance status=ready}
- 
+
 ## Lane following (LF / LFV) {#performance_lf}
+TODO: Executive decision on final form of this traveled distance
```

Comment:
The time for these TODOs is long gone - these executive decision should have been made months ago. Please decide and remove the TODO

Reply:
Good point. Have updated the pull request. We will need to briefly talk about how we want to deal with the traffic rules.